README.md

```diff
@@ -1 +1,27 @@
-# To-do-list
\ No newline at end of file
+# To-do-list
+A to-do list is a app that allow the user to organize the user's day and allow him to add a task, to remove it, to edit it and then to mark it as complet if it is
+
+# Build with
+
+HTML, CSS, Javascript, Webpack
+
+# Live Demo
+
+[link to my to-do list]()
```

Comment:
- [ ] [OPTIONAL] Kindly remove this as there is no link currently for the live version. This will make your code more clean and organized.